virttest/ip_sniffing.py

```diff
@@ -323,5 +346,35 @@ def _output_handler(self, line):
             return True
 
 
+class TShark1To2(TSharkSniffer):
+
+    """
+    TShark sniffer class for version 1.0.0 - 2.x.x.
+    """
+
+    options = ("-npi any -T fields -E separator=/s -E occurrence=f "
```

Comment:
I see that except for `bootp` and `dhcp`, other options are the same. Is it better to extract the options and pass `bootp`/`dhcp`?

Reply:
I'm not sure if it is possible, would you like to give an example? thanks.

Reply:
Em, it's possible but need to put it into the `__init__` magic method, which is not worth it. Let's keep it as it is.

Reply:
Yep, thanks.